src/util/focus-trap.spec.ts

```diff
@@ -48,7 +48,7 @@ describe('ngbFocusTrap', () => {
     });
   });
 
-  it('should save/restore focused element when autofocus is set', () => {
+  it('should focus element decorated with ngbAutofocus when created', () => {
```

Comment:
Is this tests truly verifies what it claims? I can't see any element with `ngbAutofocus` anywhere....

Reply:
Here it is https://github.com/benouat/ng-bootstrap/blob/9c946f70d9fd382ea1386233c7fa85ad661ec01d/src/util/focus-trap.spec.ts#L84

Reply:
Oh, right, I see. But still, we should have an expectation verifying that this element is actually focused, no?

Reply:
Never mind, it was just GitHub diff view not showing the whole test :-/ LGTM